services/brig/src/Brig/API/Client.hs

```diff
@@ -46,24 +48,23 @@ import qualified Brig.User.Auth.Cookie as Auth
 
 -- nb. We must ensure that the set of clients known to brig is always
 -- a superset of the clients known to galley.
-addClient :: UserId -> ConnId -> Maybe IP -> NewClient -> ExceptT ClientError AppIO Client
+addClient :: UserId -> Maybe ConnId -> Maybe IP -> NewClient -> ExceptT ClientError AppIO Client
```

Comment:
Not sure why `ConnId` is now a `Maybe`, was this meant to be just temporary... ?

Reply:
Do we always have a connId for LH devices?  Do we need one before the device is approved, but can't have one then?

Not sure...